<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>551</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Secciones Activas</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>561</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Formulario de Secciones App.</string>
        </property>
        <widget class="QLineEdit" name="lineEdit_nombre_seccion_app">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>20</y>
           <width>340</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>25</y>
           <width>130</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Nombre Sección de la App:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>115</y>
           <width>70</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Icono Sección:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_icono_seccion">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>500</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>73</y>
           <width>160</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Orden de ubicacion de la Sección:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnAgregarSeccion">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>190</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Agregar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnModificarSeccion">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>190</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Modificar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnDesactivarSeccion">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>190</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Desactivar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnLimpiarFormulario">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>190</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Limpiar</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="Tabla_secciones_activas">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>220</y>
           <width>520</width>
           <height>300</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>id</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nombre Sección</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Orden Ubicación</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ubicacion Logo</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
        <widget class="QPushButton" name="btnCerrar">
         <property name="geometry">
          <rect>
           <x>494</x>
           <y>10</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border : none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../assets/iconos/cerrar.png</normaloff>../assets/iconos/cerrar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="btnIconoSeccion">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>110</y>
           <width>70</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Buscar Icono</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_icono_seccion">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>50</y>
           <width>104</width>
           <height>104</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>104</width>
           <height>104</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid gray;
border-radius: 50px;
min-width: 100px;
min-height: 100px;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_orden_seccion">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>70</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
        <widget class="QPushButton" name="btnEliminarSeccion">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>190</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Eliminar</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Secciones Inactivas</string>
    </attribute>
    <widget class="QTableWidget" name="Tabla_secciones_inactivas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>501</width>
       <height>371</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>id Usuario</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>D.N.I</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Domicilio</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Localidad</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Provincia</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Teléfono</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Usuario</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Contraseña</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ruta Foto</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rol</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnReactivarSeccion">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>400</y>
       <width>141</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Reactivar Sección</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_nombre_seccion_app</tabstop>
  <tabstop>spinBox_orden_seccion</tabstop>
  <tabstop>btnIconoSeccion</tabstop>
  <tabstop>lineEdit_icono_seccion</tabstop>
  <tabstop>btnAgregarSeccion</tabstop>
  <tabstop>btnModificarSeccion</tabstop>
  <tabstop>btnEliminarSeccion</tabstop>
  <tabstop>btnDesactivarSeccion</tabstop>
  <tabstop>btnLimpiarFormulario</tabstop>
  <tabstop>Tabla_secciones_activas</tabstop>
  <tabstop>btnCerrar</tabstop>
  <tabstop>Tabla_secciones_inactivas</tabstop>
  <tabstop>btnReactivarSeccion</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
